File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        map = {}
        ret = []
        for x in nums:
            if x not in map:
                map[x]=1
            else:
                map[x]+=1

        map = sorted(map.items(), key = lambda x:x[1],reverse=True)
        for key,_ in map:
            ret.append(key)
            k-=1
            if k==0: break
        return ret

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int ret = 0, temp = 0;
        for(auto x : nums) {
            if(x==1) temp++;
            else {ret = (temp>ret ? temp : ret); temp=0;}
            cout<<"temp now: "<<temp<<'\n';
        }
        return (temp>ret ? temp : ret);
    }
};